pkg/client/cli/cloud/message.go

```diff
@@ -31,18 +32,18 @@ type messageCache struct {
 }
 
 // newMessageCache returns a new messageCache, initialized from the users' if it exists.
-func newMessageCache(ctx context.Context) (*messageCache, error) {
+func newMessageCache(ctx context.Context) *messageCache {
```

Comment:
I had no idea this thing existed, I would have thought it was in pro. I guess this change only mitigates the issue.